frontend/integration-tests/views/service-catalog.view.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-undef, no-unused-vars */
+
+import {browser, $, ExpectedConditions as until, $$, by} from 'protractor';
+
+const cscLinks = $$('.co-cluster-service-class-link');
```

Comment:
Is it possible to use the existing `rowForName` instead of adding a new class?

https://github.com/openshift/console/blob/master/frontend/integration-tests/views/crud.view.ts#L19

Reply:
No, service classes do not use the standard resource rows